lib/publiccloud/ec2.pm

```diff
@@ -253,4 +253,21 @@ sub start_instance
     $instance->public_ip($public_ip);
 }
 
+sub get_instance_type
```

Comment:
IMHO this function can be replaced with `$self->describe_instance($instance)->{InstanceType}` which will be also more descriptive for code reader